src/job/job.spec.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-env jest */
+
+import { omit } from "lodash";
+
+import config from "../_config";
+import { xo, testWithOtherConnection } from "../util";
+
+const ADMIN_USER = {
+  email: "admin2@admin.net",
+  password: "admin",
+  permission: "admin",
+};
+
+const DEFAULT_JOB = {
+  name: "jobTest",
+  timeout: 2000,
+  type: "call",
+  key: "snapshot",
+  method: "vm.snapshot",
+};
+
+describe("job", () => {
+  describe(".create() :", () => {
+    it("creates a new job", async () => {
+      const userId = await xo.createUser(ADMIN_USER);
+      const { email, password } = ADMIN_USER;
+      const job = { ...DEFAULT_JOB };
```

Comment:
inline it.

Reply:
Why not add it to the `DEFAULT_JOB`?

Reply:
because `config` is undefined during initialization of global constants.